### PR TITLE
Edit: Fix typo in /docs-v1 and /resources

### PR DESCRIPTION
**Pull request by jenpark-getty**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [ ] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

**N/A**

### Please describe the goal of this pull request and the changes that were made.

**Fix typo in /docs-v1 and /resources**

### CODE REVIEW

**Review by geealbers — APPROVED** [review]
Thanks for catching those, Jenny!

I've approved this PR (pull request) with your fixes. And whenever Erin or I approve a PR you can go ahead and merge it in yourself, and then always delete the branch if the repo isn't set to auto-delete. It will tell you after you hit the green Merge button.

**Comment by jenpark-getty** [comment]
Got it, thanks Greg!

**Pull request merged**
